extensions/farcaster/src/components/CastDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import { Action, ActionPanel, Detail, Icon } from '@raycast/api';
+import { Cast } from '../utils/types';
+import { getUserIcon, getCastUrl } from '../utils/helpers';
+
+export default function CastDetails({ cast }: { cast: Cast }) {
+  const hasFrame = cast.frames && cast.frames[0]?.image;
+  const imageEmbeds = cast.embeds.filter(
+    (embed): embed is { url: string } => 'url' in embed && embed.url.startsWith('https://i.imgur.com/'),
+  );
+  const raycastWidth = imageEmbeds.length === 1 ? 300 : 180;
+  const imagesPerRow = imageEmbeds.length === 1 ? 1 : 2;
+
+  const image = imageEmbeds
+    .map((embed, index) => {
+      const separator = index % imagesPerRow === imagesPerRow - 1 ? '\n\n' : ' ';
+      return `![image](${embed.url}?raycast-width=${raycastWidth})${separator}`;
+    })
+    .join('');
+
+  const frame = hasFrame ? `![image](${cast.frames[0]?.image})` : '';
+  {
+    /* support for cast embeds */
+  }
+  const markdown = `
+  ${cast.text}
+  ${image}
+  ${frame}
+  `;
+
+  return (
+    <Detail
+      markdown={markdown}
+      metadata={<CastMetadata cast={cast} />}
+      actions={
+        <ActionPanel>
+          <Action.OpenInBrowser
+            title="View Cast in Browser"
+            shortcut={{ modifiers: ['ctrl'], key: 'enter' }}
```

Comment:
I just tested it in another extension since I don't have any credentials - it should work just fine

![Raycast17042024-aypYDnOW png@2x](https://github.com/raycast/extensions/assets/38033107/291aa390-3547-40bf-b313-bbbb623062bf)

Reply:
If you look at the img you posted the details triggers both from hitting enter (in the main menu) and ctrl+enter (in the action panel). I was hoping I could set it up such that it had to be ctrl+enter for the main menu as well.

Reply:
That's how Raycast works, default action will always have `enter`